todo-app/src/state.js

```diff
@@ -1,11 +1,36 @@
 import React from 'react'
 
-export default class State extends React.Component{
+export default class Status extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+          done: this.props.status
+        }
+        this.changedStatus = this.changedStatus.bind(this);
+      }
+      changedStatus(){
```

Comment:
This is not used in the file yet? When do you want to call this function? When checkbox input changes right?